packages/backend/src/utils.ts

```diff
@@ -8,33 +8,30 @@ import {
     ProcessAttestationsProof,
 } from '@unirep/contracts'
 import { DB } from 'anondb'
+import { UNIREP, UNIREP_ABI, DEFAULT_ETH_PROVIDER } from './constants'
```

Comment:
I think we can use abi from `@unirep/contracts`

Reply:
Yep, this is changed in #71